README.md

```diff
@@ -38,7 +38,9 @@ export default class extends React.Component {
 
 ### Running demo locally
 
-The demo can be launched on your local machine via `webpack-dev-server`. Once you have clone this repo locally, run the following:
+The demo can be launched on your local machine via `webpack-dev-server`. Make sure you are running the node version 9.11 or earlier. Once you have clone this repo locally, run the following:
```

Comment:
I know you didn't write this part but I found a typo in `Once you have clone...` it should be `Once you have cloned...`

Reply:
I would also say `...running node version 9.11` and remove the `the`

Reply:
Thanks.I will change that today.